Zend/zend_compile.c

```diff
@@ -2813,6 +2815,32 @@ static void zend_verify_list_assign_target(zend_ast *var_ast, zend_bool old_styl
 }
 /* }}} */
 
+static zend_bool zend_set_list_assign_reference(zend_ast *ast) { /* {{{ */
+	zend_bool has_reference = 0;
+	zend_ast_list *list = zend_ast_get_list(ast);
+	uint32_t i;
+
+	for (i = 0; i < list->children; ++i) {
+		zend_ast *elem_ast = list->child[i];
+		if (elem_ast) {
+			zend_ast *var_ast = elem_ast->child[0];
+
+			if (elem_ast->kind == ZEND_AST_ARRAY_ELEM && elem_ast->attr) {
```

Comment:
I don't believe so, I tried thinking of what other sub-container of variables could exist within list() syntax.  Or, at least, I'm not familiar with any other.